### PR TITLE
Removing httpretty

### PR DESCRIPTION
**Pull request by josenavas**
This PR removes the mock REST api using httpretty and uses the local installation of Qiita to test the plugin.

Currently I've only modified one test to check if it works as expected, I will modify the other ones once I know this works.

### CODE REVIEW

**Comment by josenavas** [comment]
This PR is ready for review - however, tests will fail until https://github.com/biocore/qiita/pull/1862 is merged.

**Comment by antgonza** [comment]
@josenavas, could you let us know when tests pass? Thanks.

**Comment by josenavas** [comment]
Passed!

File: qiita_client/tests/test_qiita_client.py [hunk 1]
```diff
@@ -158,80 +132,69 @@ def test_patch_value_error(self):
         # Add, replace or test
         with self.assertRaises(ValueError):
             self.tester.patch(
-                '/qiita_db/artifacts/1/filepaths/', 'add', '/html_summary/',
+                '/qiita_db/artifacts/1/', 'add', '/html_summary/',
```
Comment (antgonza) [inline qiita_client/tests/test_qiita_client.py]: So filepaths is not being tested anymore?

**Comment by antgonza** [comment]
1 question.

File: qiita_client/tests/test_qiita_client.py (hunk 1 above)
Comment (josenavas) [inline qiita_client/tests/test_qiita_client.py]: The filepaths endpoint no longer exists. In any case, this is just checking the `patch` functionality of the qiita_client, so the only thing that I needed is an endpoint that allows patching.

Comment (antgonza) [inline qiita_client/tests/test_qiita_client.py]: ok, thanks.

**Pull request merged**